source/github-helpers/selectors.ts

```diff
@@ -78,7 +78,10 @@ export const openPrsListLink = [
 	`,
 	// React view
 	css`
-		li[role="listitem"] a[data-hovercard-url*="/pull"]
+		li[role="listitem"]:has(
+			.octicon-git-pull-request.color-fg-open,
+			.octicon-git-pull-request-draft
+		) a[data-hovercard-url*="/pull"]
```

Comment:
I think you just need to make the selector more specific without `has`

```suggestion
		li[role="listitem"] h3 a[data-hovercard-url*="/pull"]
```

Or even but just using the new `[data-testid="issue-pr-title-link"]` on its own